packages/bot-skeleton/src/scratch/blocks/Advanced/Functions/procedures_callnoreturn.js

```diff
@@ -55,7 +56,7 @@ Blockly.Blocks.procedures_callnoreturn = {
      * @this Blockly.Block
      */
     onchange(event) {
-        if (!this.workspace || this.workspace.isFlyout) {
+        if (!this.workspace || Blockly.derivWorkspace.isFlyout_) {
```

Comment:
if this is a readonly value, is there any setter function that sets this? if there is no setter can you find out what sets this value internally within `Blockly`

Reply:
We are internally using this isFlyout_ property and it is not a private property from blockly. I have Just changed the naming to `isFlyoutVisible`.